JustSaying.AwsTools/SqsNotificationListener.cs

```diff
@@ -88,7 +80,7 @@ public SqsNotificationListener WithMessageProcessingStrategy(IMessageProcessingS
             }
 
             Subscribers.Add(new Subscriber(typeof(T)));

```

Comment:
That list is used for the interrogation feature - to allow clients to show the current state of JustSaying. We use it at JUST EAT so that we can see, at run time, what events a component has subscribed to / what events it can raise.